student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super()
```

Comment:
- [ ] We will need to initialize superclass params as well, it should be something like the following:
```suggestion
  def initialize(age, classroom, name = "Unknown", parent_permission = true) 
    super(age, name, parent_permission)
```